File: java-program-design/src/test/java/chapter/five/one/two/one/ClassTestTest.java
package chapter.five.one.two.one;

import org.example.MockTest;
import org.junit.jupiter.api.Test;

import java.util.Arrays;

class ClassTestTest {

    @Test
    void maintest() {
        System.out.println(Arrays.toString(MockTest.MockTest("java.util.Date\n", () -> ClassTest.main(new String[]{}))));
    }
}

File: java-program-design/src/test/kotlin/chapter/five/one/two/one/ClassTestTest.kt
package chapter.five.one.two.one

import com.github.masx200.learn_java_kotlin_programming.utils.MockTest
import org.junit.jupiter.api.Test

internal class ClassTestTest {
    @Test
    fun maintest() {
        println(MockTest.MockTest("java.util.Date\n") { ClassTest.main(arrayOf()) }
            .contentToString())
    }
}